tests/func/test_import.py

```diff
@@ -67,6 +68,23 @@ def test_pull_imported_stage(dvc_repo, erepo):
     assert os.path.isfile(dst_cache)
 
 
+def test_pull_imported_directory_stage(dvc_repo, erepo):
+    src = erepo.DATA_DIR
+    dst = erepo.DATA_DIR + "_imported"
+    stage_file = erepo.DATA_DIR + "_imported.dvc"
```

Comment:
You could use

```suggestion
    stage_file = dst + ".dvc"
```

or just use stage object returned from imp .